AndrewTehan/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# this class is my realisation of Hash
+class MyHash
+  include Enumerable
+
+  def initialize
+    @keys_values = []
+  end
+
+  def [](key)
+    pair(key)[1]
+  end
+
+  def []=(key, value)
+    if self[key]
+      @keys_values.find { |k, _v| k == key }[1] = value
+    else
+      @keys_values.push([key, value])
+    end
```

Comment:
Could you pls change this according to [the article](https://anthonygharvey.com/guard-clauses-vs-nested-conditionals)?